src/main/scala/mesosphere/marathon/tasks/OfferMatcher.scala

```diff
@@ -205,8 +214,15 @@ class IterativeOfferMatcher @Inject() (
         launchedTasks += tasks.size()
       }
       else {
-        log.debug("declining offer {}", offer.getId)
-        driver.declineOffer(offer.getId)
+        iterativeOfferMatcherConfig.rejectOfferDuration.get match {
```

Comment:
No, it's not exactly, even though similar. It is a `ScallopOption[Long]`. The `.get` method actually returns an `Option[Long]`